<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CleaningApp</class>
 <widget class="QMainWindow" name="CleaningApp">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>451</width>
    <height>396</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Data Cleaning Tool</string>
  </property>
  <property name="styleSheet">
   <string notr="true">MainWindow{
Background:qlineargradient(spread:pad, x1:0.949, y1:1, x2:1, y2:0, stop:0 rgba(0, 0, 0, 255), stop:1 rgba(255, 255, 255, 255));
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label_6">
          <property name="font">
           <font>
            <family>Palatino Linotype</family>
           </font>
          </property>
          <property name="text">
           <string>Browse for the source URL of the video:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton">
          <property name="font">
           <font>
            <family>MV Boli</family>
           </font>
          </property>
          <property name="text">
           <string>Browse</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="font">
         <font>
          <family>Palatino Linotype</family>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">
ui-&gt;label-&gt;setStyleSheet(&quot;{color: blue}&quot;);
</string>
        </property>
        <property name="text">
         <string>&lt;empty&gt;</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_7">
          <property name="font">
           <font>
            <family>Palatino Linotype</family>
           </font>
          </property>
          <property name="text">
           <string>Browse for the destination URL folder:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_3">
          <property name="font">
           <font>
            <family>MV Boli</family>
           </font>
          </property>
          <property name="text">
           <string>Browse</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="font">
         <font>
          <family>Palatino Linotype</family>
         </font>
        </property>
        <property name="text">
         <string>&lt;empty&gt;</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="2" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QLabel" name="label_5">
            <property name="font">
             <font>
              <family>Palatino Linotype</family>
             </font>
            </property>
            <property name="text">
             <string>Images/Second:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="FrameCountBox">
            <property name="font">
             <font>
              <family>Palatino Linotype</family>
             </font>
            </property>
            <property name="maximum">
             <number>60</number>
            </property>
            <property name="value">
             <number>1</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="label_3">
            <property name="font">
             <font>
              <family>Palatino Linotype</family>
             </font>
            </property>
            <property name="text">
             <string>Horizontal resolution:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="XResBox">
            <property name="font">
             <font>
              <family>Palatino Linotype</family>
             </font>
            </property>
            <property name="minimum">
             <number>100</number>
            </property>
            <property name="maximum">
             <number>1920</number>
            </property>
            <property name="value">
             <number>720</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QLabel" name="label_4">
            <property name="font">
             <font>
              <family>Palatino Linotype</family>
             </font>
            </property>
            <property name="text">
             <string>Vertical resulotion:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="YResBox_3">
            <property name="font">
             <font>
              <family>Palatino Linotype</family>
             </font>
            </property>
            <property name="minimum">
             <number>100</number>
            </property>
            <property name="maximum">
             <number>1080</number>
            </property>
            <property name="value">
             <number>480</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item row="3" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <item>
       <widget class="QTextBrowser" name="textBrowser">
        <property name="font">
         <font>
          <family>Palatino Linotype</family>
         </font>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_2">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>23</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>MV Boli</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">pushButton_2{
Background:rgb(85, 0, 0);
}</string>
        </property>
        <property name="text">
         <string>RUN</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>451</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
